test/storage/2019-07-07/azblob/zz_generated_appendblob.go

```diff
@@ -114,18 +113,18 @@ func (client *appendBlobClient) appendBlockHandleResponse(resp *azcore.Response)
 		result.LastModified = &lastModified
 	}
 	if val := resp.Header.Get("Content-MD5"); val != "" {
-		contentMd5, err := base64.StdEncoding.DecodeString(val)
+		contentMD5, err := base64.StdEncoding.DecodeString(val)
 		if err != nil {
 			return AppendBlobAppendBlockResponse{}, err
 		}
-		result.ContentMD5 = &contentMd5
+		result.ContentMD5 = &contentMD5
 	}
 	if val := resp.Header.Get("x-ms-content-crc64"); val != "" {
-		contentCrc64, err := base64.StdEncoding.DecodeString(val)
+		xMSContentCRC64, err := base64.StdEncoding.DecodeString(val)
```

Comment:
This is failing the unexported lowercasing for the first section of the var

Reply:
I opted to leave this as-is given that it's an internal var (if it were a param I'd fix it).

Reply:
I can fix if you feel strongly about it.

Reply:
No no, it's fine was just worried if it leaked to other fields. It's fine to leave it on the internal var, we can fix it later if we want to.